src/shipit_dashboard/migrations/versions/3a54bd40145f_.py

```diff
@@ -56,7 +56,12 @@ def upgrade():
     ### end Alembic commands ###
 
     # Empty analysis
-    op.execute("TRUNCATE TABLE shipit_bug_analysis CASCADE")
+    try:
+        # Postgresql
+        op.execute("TRUNCATE TABLE shipit_bug_analysis CASCADE")
+    except:
+        # Sqlite
+        op.execute("DELETE FROM shipit_bug_analysis")
```

Comment:
i created this issue which we can solve at the later point to always run against postgresql
https://github.com/mozilla-releng/services/issues/95